contracts/SimpleBank.sol

```diff
@@ -1,49 +1,48 @@
 /*
-    This exercise has been updated to use Solidity version 0.6.12
-    Breaking changes from 0.5 to 0.6 can be found here: 
-    https://solidity.readthedocs.io/en/v0.6.12/060-breaking-changes.html
+    This exercise has been updated to use Solidity version 0.8.5
+    See the latest Solidity updates at
+    https://solidity.readthedocs.io/en/latest/080-breaking-changes.html
 */
-
-pragma solidity ^0.5.0;
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.5.16 <0.9.0;
 
 contract SimpleBank {
 
     //
     // State variables
     //
     
-    /* Fill in the keyword. Hint: We want to protect our users balance from other contracts*/
-    mapping (address => uint) balances;
+    // Fill in the keyword. 
+    // Hint: We want to protect our users balance from other contracts
+    mapping (address => uint) balances ;
     
-    /* Fill in the keyword. We want to create a getter function and allow contracts to be able to see if a user is enrolled.  */
+    // Fill in the keyword. 
+    // Hint: We want to create a getter function and allow contracts to be able
+    // to see if a user is enrolled.
     mapping (address => bool) enrolled;
 
-    /* Let's make sure everyone knows who owns the bank. Use the appropriate keyword for this*/
-    address owner;
+    // Let's make sure everyone knows who owns the bank. 
+    // Hint: Use the appropriate keyword for this
+    address owner = msg.sender;
     
     //
     // Events - publicize actions to external listeners
     //
     
-    /* Add an argument for this event, an accountAddress */
+    // Add an argument for this event, an accountAddress
     event LogEnrolled();
 
-    /* Add 2 arguments for this event, an accountAddress and an amount */
+    // Add 2 arguments for this event, an accountAddress and an amount
     event LogDepositMade();
 
-    /* Create an event called LogWithdrawal */
-    /* Add 3 arguments for this event, an accountAddress, withdrawAmount and a newBalance */
-
+    // Create an event called LogWithdrawal
+    // Hint: it should take 3 arguments: an accountAddress, withdrawAmount and a newBalance 
+    event LogWithdrawal();
 
     //
     // Functions
     //
 
-    /* Use the appropriate global variable to get the sender of the transaction */
```

Comment:
Curious about this, you didn't want to give the student too much initially?

Reply:
Judgement call, I think the slack/office hours support should kick in